milestone2.ts

```diff
@@ -83,32 +83,64 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
-
-//console.log(findAverage(userData));
+let findAverage: GetNumber = (data) => { // Define lambda function here
+  let averageAge = 0;
+  let count = 0;
+  for(let i=0; i<data.length; i++){
+    averageAge += data[i].age;
+    count++;
+  }
+  return (averageAge / count);
+} 
+
+// console.log(findAverage(userData));
 
 // Question 2: Data Handling
 /* Use data handling function(s) to find the first index of someone from San Francisco (SF).
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = (data) => {  // Code here
+  for(let i=0; i<data.length; i++){
+    if (data[i].city === "SF") {
+      return i;
+    }
+  }
+
+  return -1;
```

Comment:
Can you redo this question using data handling functions like specified in the question. If you need a reference on data handling functions, look at slide 9 of [Bootcamp session 4](https://docs.google.com/presentation/d/1Jdvx7Ji4n3m6jODDX2N_gL5BO-ZTEguSNa_drSd5s1g/edit?usp=sharing)